myko/src/lib/components/DiyView.svelte

```diff
@@ -50,14 +59,30 @@
         <p>
           Bra att veta om det är en aktivitet för flera och om vi har frågor eller vill följa upp.
         </p>
+
         <input type="text" name="name" />
       </div>
 
       <div>
-        <input type="submit" value="Notera aktiviteten" required class="btn secondary" />
+        <input
+          type="submit"
+          value="Notera aktiviteten"
+          required
+          class="btn"
+          action="h1"
```

Comment:
There is a "Svelte-way" to doing this, avoiding hardcoding the element and only scrolling to the error message element if it is shown (so not when form is successfully posted). I tested it [here](https://github.com/Relationsinstitutet/myko/pull/99) and seems to work nicely.

Reply:
Great that's so cool! Will have to read up on those bindings

Reply:
Hm realized now on testing that the error message doesn't go away. If you would first not fill the form in correctly, but then upon correction do it right, you end up with both error and success messages.  
Not sure how to fix this

Reply:
Nice catch! I made another PR fixing that: https://github.com/Relationsinstitutet/myko/pull/101